main.lua

```diff
@@ -98,22 +99,34 @@ local activeRole = "803685296083828736"
 
 local function handlePoints(message)
   -- Ignores bots and DMs
-  if not message.guild or message.author.bot then return end
+  if not message.guild or message.author.bot then
+    return
+  end
   -- Ignore the #memes and #bots channels.
-  if message.channel.id == "820869096894890015" or message.channel.id == "641655510692790286" then return end
+  if message.channel.id == "820869096894890015" or message.channel.id == "641655510692790286" then
+    return
+  end
```

Comment:
There is a `collapse_simple_statement` option that we could set to `ConditionalOnly`.
but I don't like collapsed statements of any kind and agree with the default of `Never`.